outputs.tf

```diff
@@ -1,15 +1,15 @@
 output "bucket_domain_name" {
-  value       = "${aws_s3_bucket.default.bucket_domain_name}"
+  value       = "${aws_s3_bucket.default.*.bucket_domain_name}"
```

Comment:
let's use `splat+join` pattern here since this is a list now `aws_s3_bucket.default.*.bucket_domain_name`.

```hcl
"${join("", aws_s3_bucket.default.*.bucket_domain_name)}"
```

although `aws_s3_bucket.default.*.bucket_domain_name` (without `join`) will work in many cases, in some more complex configuration it will break.